tox.ini

```diff
@@ -36,6 +36,17 @@ commands =
         docs \
         build/docs/python
 
+[testenv:ansible]
+skip_install = true
+skipsdist = true
+changedir = devops/ansible-role-girder
+passenv = DOCKER_*
```

Comment:
These are set from circle?

Reply:
Yes, and the `setup_remote_docker` CircleCI sets up a remote Docker daemon to run commands.